pysr/export_latex.py

```diff
@@ -0,0 +1,34 @@
+"""Functions to help export PySR equations to LaTeX."""
+import re
+
+
+def set_precision_of_constants_in_string(s, precision=3):
+    """Set precision of constants in string."""
+    constants = re.findall(r"\b[-+]?\d*\.\d+|\b[-+]?\d+\.?\d*", s)
+    for c in constants:
+        reduced_c = "{:.{precision}g}".format(float(c), precision=precision)
+        s = s.replace(c, reduced_c)
```

Comment:
I'm realizing this might produce some bugs, if a constant is just something like `"1"` - then all the `1`s would be replaced...

Is there a better usage of `re` here?

Reply:
Should just write a custom `sympy.LatexPrinter` instead...